packages/ui/src/Toggle.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { FC, FocusEvent, ChangeEvent, AriaAttributes, useRef } from 'react'
+import cn from 'classnames'
+import { errorId } from './Error'
+// import { Error, errorId } from './Error'
+import { Help, helpTooltipId } from './Help'
+import { useUID } from 'react-uid'
+import { DataTestProp } from '@hazelcast/helpers'
+import styles from './Toggle.module.scss'
+
+export type ToggleCoreProps = {
+  name: string
+  value?: string
+  checked?: boolean
+  onBlur?: (e: FocusEvent<HTMLInputElement>) => void
+  onChange?: (e: ChangeEvent<HTMLInputElement>) => void
+  error?: string
+
+  label: string | React.ReactNode
+  helperText?: string
+  disabled?: boolean
+  className?: string
+  classNameLabel?: string
+}
+
+export type ToggleProps = ToggleCoreProps & DataTestProp
+
+export const Toggle: FC<ToggleProps> = ({
+  name,
+  value,
+  onChange,
+  onBlur,
+  error,
+  checked,
+  label,
+  helperText,
+  disabled,
+  className,
+  classNameLabel,
+  'data-test': dataTest,
+}) => {
+  const inputRef = useRef<HTMLInputElement>(null)
+  const id = useUID()
+
+  let ariaChecked: AriaAttributes['aria-checked'] = checked ? 'true' : 'false'
```

Comment:
nit: why do we need to extract this variable? Couldn't we simply pass 'checked' to the aria-checked prop of the input?

Reply:
On second thought, why do we need aria-checked at all? We already have checked prop, shouldn't it be enough?

Reply:
hmm, i based in on Checkbox without giving it much thought. will check it out...

Reply:
For checkbox it makes sense because checkbox could be `indeterminate`. It is not the issue with the toggle

Reply:
yep, on a second look it's unnecessary. less LOC is best LOC. pushed the code. (ps. it's awesome to work with this level of peer reviews, thx a lot @aigoncharov, keep em coming!!)